### PR TITLE
sic: init at 1.2

### PR DESCRIPTION
**Pull request by dbrock**
My first attempt at creating a Nix package :-)

### CODE REVIEW

**Comment by lethalman** [comment]
Hi. You don't need a builder.sh. Remove it, you can set `makeFlags` directly in the nix derivation.

File: pkgs/applications/networking/irc/sic/default.nix [hunk 1]
```diff
@@ -0,0 +1,18 @@
+{ stdenv, fetchurl }:
+
+stdenv.mkDerivation rec {
+  name = "sic-${version}";
+  version = "1.2";
+
+  builder = ./builder.sh;
+  src = fetchurl {
+    url = "http://dl.suckless.org/tools/sic-${version}.tar.gz";
+    md5 = "eb099e790c3ce7866df50d3fe1a52c25";
```
Comment (vcunat) [inline pkgs/applications/networking/irc/sic/default.nix]: Please, use sha256 instead of md5. /cc #4491.

**Comment by vcunat** [comment]
BTW, if you're new to github, you can force-push to the same branch to update the corresponding pull request.

I assume you're using the package now, so it's tested a bit?

**Comment by dbrock** [comment]
@lethalman @vcunat Ah, I was hoping there would be a way to avoid the `builder.sh` script --- nice!

I switched to SHA-256 as well.

**Comment by lethalman** [comment]
Travis built. Thanks and welcome :)

**Pull request merged**